From 59d8ce80ac4e715f4d02c072e9a8f161360b8667 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 13 Dec 2022 15:42:41 +0100
Subject: [PATCH] arm64: dts: st: remove SDR104 and HS200 modes on
 stm32mp257f-valid3

Some instabilities have been seen on SD-cards using SDR104 mode on
stm32mp257f-valid3. Temporary remove the feature.
The HS200 mode on eMMC which also uses delay block and feedback clock
is also removed till the issue is not understood.

Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I9c14464191ea8fc76ebefee969fd69d978e2ad6f
---
 arch/arm64/boot/dts/st/stm32mp257f-valid3-daughter.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-valid3-daughter.dts b/arch/arm64/boot/dts/st/stm32mp257f-valid3-daughter.dts
index 1e0fad95ec9c..a4e399af1b16 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-valid3-daughter.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-valid3-daughter.dts
@@ -246,7 +246,6 @@ &sdmmc1 {
 	sd-uhs-sdr25;
 	sd-uhs-sdr50;
 	sd-uhs-ddr50;
-	sd-uhs-sdr104;
 	status = "okay";
 };
 
@@ -263,7 +262,6 @@ &sdmmc2 {
 	vmmc-supply = <&v3v3>;
 	vqmmc-supply = <&scmi_vddio2>;
 	mmc-ddr-1_8v;
-	mmc-hs200-1_8v;
 	status = "okay";
 };
 
-- 
2.25.1

